### PR TITLE
bug fix shuai

### PR DESCRIPTION
**Pull request by shuailiu-zingbox**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
In case you are a technological partner:
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestoservicenow/sendbulkpanwiotdevicestoservicenow.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 comment: This script gets all available devices from the IoT cloud and updates/creates
   new endpoint with custom attributes on ServiceNow
 commonfields:
-  id: send_bulk_panw_iot_devices_to_servicenow
+  id: send_bulk_panw_iotices_to_servicenow
```
Comment (shuailiu-zingbox) [inline Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestoservicenow/sendbulkpanwiotdevicestoservicenow.yml:4]: iotices typo need to change

File: Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestoservicenow/sendbulkpanwiotdevicestoservicenow.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   version: -1
 dockerimage: demisto/python3:3.8.6.12176
 enabled: true
-name: send_bulk_panw_iot_devices_to_servicenow
+name: send_bulk_panw_iotices_to_servicenow
```
Comment (shuailiu-zingbox) [inline Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestoservicenow/sendbulkpanwiotdevicestoservicenow.yml:8]: same here

File: Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestosiem/sendbulkpanwiotdevicestosiem.yml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 comment: gets all available devices from the IoT cloud and sends it to the SIEM server
 commonfields:
-  id: send_bulk_panw_iot_devices_to_siem
+  id: send_bulk_panw_iotices_to_siem
```
Comment (shuailiu-zingbox) [inline Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestosiem/sendbulkpanwiotdevicestosiem.yml:3]: same here

File: Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestosiem/sendbulkpanwiotdevicestosiem.yml [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
   version: -1
 dockerimage: demisto/python3:3.8.6.12176
 enabled: true
-name: send_bulk_panw_iot_devices_to_siem
+name: send_bulk_panw_iotices_to_siem
```
Comment (shuailiu-zingbox) [inline Packs/PaloAltoIot3rdPartyBase/Scripts/sendbulkpanwiotdevicestosiem/sendbulkpanwiotdevicestosiem.yml:7]: same here

File: Packs/PaloAltoIot3rdPartyBase/Scripts/sendpanwiotdevicestoise/sendpanwiotdevicestoise.yml [hunk 5]
```diff
@@ -6,11 +6,11 @@ args:
 comment: This script takes in custom attributes from PANW IoT cloud and create or
   updates endpoints in ISE
 commonfields:
-  id: send_panw_iot_devices_to_ise
+  id: send_panw_iotices_to_ise
```
Comment (shuailiu-zingbox) [inline Packs/PaloAltoIot3rdPartyBase/Scripts/sendpanwiotdevicestoise/sendpanwiotdevicestoise.yml:9]: same here

File: Packs/PaloAltoIot3rdPartyBase/Scripts/sendpanwiotdevicestoise/sendpanwiotdevicestoise.yml [hunk 6]
```diff
@@ -6,11 +6,11 @@ args:
...
   version: -1
 dockerimage: demisto/python3:3.8.6.12176
 enabled: true
-name: send_panw_iot_devices_to_ise
+name: send_panw_iotices_to_ise
```
Comment (shuailiu-zingbox) [inline Packs/PaloAltoIot3rdPartyBase/Scripts/sendpanwiotdevicestoise/sendpanwiotdevicestoise.yml:13]: same here

File: Packs/PaloAltoIot3rdPartyBase/Playbooks/panw_iot_ise_quarantine_playbook.yml [hunk 7]
```diff
@@ -21,12 +21,12 @@ tasks:
     skipunavailable: false
     task:
```
Comment (dawar-zingbox) [inline Packs/PaloAltoIot3rdPartyBase/Playbooks/panw_iot_ise_quarantine_playbook.yml:22]: this one should not be touched. Looks like you are using an old version of code here again

**Pull request closed without merging**